modules/mongodb/mongodb.go

```diff
@@ -125,10 +125,19 @@ func (c *MongoDBContainer) ConnectionString(ctx context.Context) (string, error)
 	if err != nil {
 		return "", err
 	}
+
```

Comment:
suggestion: this could fail depending on the values, lets change it to use url / net functions to ensure the value is valid, for example:
```go
	u := url.URL{
		Scheme: "mongodb",
		Host:   net.JoinHostPort(host, port),
	}

	if c.username != "" || c.password != "" {
		u.User = url.UserPassword(c.username, c.password)
	}

	if c.replicaSet != "" {
	        v := url.Values{}
		v.Add("replicaSet", c.replicaSet)
	        u.RawQuery = v.Encode()
	}

        return  u.String(), nil
```

Reply:
Updated the code by following the suggestion.